cmd/rancher-desktop-guestagent/main.go

```diff
@@ -198,11 +198,15 @@ func main() {
 					"Valid options are 0.0.0.0 and 127.0.0.1.", *k8sServiceListenerAddr)
 			}
 
+			// listenerOnlyMode represents when iptables is enabled and
+			// privileged services is disabled this can indicate a non-admin
+			// installation of default network which requires listeners only.
```

Comment:
```suggestion
			// listenerOnlyMode represents when iptables is enabled and
			// privileged services is disabled; this can indicate a non-admin
			// installation of default network which requires listeners only.
```
I don't actually understand what that second part is saying; please rephrase.